src/pages/Shop/Shop.js

```diff
@@ -1,5 +1,113 @@
 import React from 'react';
+import styled from 'styled-components';
+import LeftContainer from './LeftSide/LeftContainer';
+import RightContianer from './RightSide/RightContianer';
+import TopsideContainer from './Topside/TopsideContainer';
 
 export default function Shop() {
-  return <div>Shop</div>;
+  // window.addEventListener('scroll', () => console.log('🌀'));
+  const BrandButton = [
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '조던',
+    },
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '나이키',
+    },
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '컨버스',
+    },
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '반스',
+    },
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '반스',
+    },
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '반스',
+    },
+    {
+      Image_url:
+        'https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80"',
+      brand: '반스',
+    },
+  ];
+  return (
+    <WholeShop>
+      <ShopWrapper>
+        <TopsideContainer />
+        <ShoeBrandBtn>
+          {BrandButton &&
+            BrandButton.map((brandBox, idx) => (
```

Comment:
```suggestion
          {brandButton.map((brandBox, idx) => (
```
이렇게 하더라도 문제 없을거 같은데 왜일까요? 생각해보시고 이유 댓글로 남겨주세요~

Reply:
* &&연산자를 쓰는 의미는 항상 true일때 정한 것이 
          나오도록하는것인데 이 조건이 필요 없다는 것 그리고 state값이 null 일때 와 
        데이터의 []을 넣어줄때의 차이 
        처음에 라이프사이클에 의해서 처음 랜더할때 state값에 null 을 널어주게 되면 undifined가 뜨게 되는데 여기서
        처음 랜더될때 오류가 뜨게되고 그걸 해결하기 위해  안전장치로 &&연산자로 
        조건을 넣어주게되서 항상 참으로 되게해서 랜더가 되게한거고 만약 &&연산자를 없게 해결되게하면 
        state값에 null이 아닌 빈 배열값이나 객체값을 받게 해서 처음 받을때 undifined가 안뜨게
        하면된다. 라고 정의를 해보았지만 부족한점이 많은것같습니다